system/lib/libc/emscripten_memcpy.c

```diff
@@ -8,21 +8,25 @@
 #include "libc.h"
 #include "emscripten_internal.h"
 
+#if !defined(__wasm_bulk_memory__)
+#error "This file must be compile with bulk memory enabled"
+#endif
```

Comment:
I think we can just drop these 3 lines.  I'm not sure what they achieve.

Reply:
Ah yes, they were more useful when I had inline asm in this file.